tools/db/replicateDbs.py

```diff
@@ -66,6 +68,12 @@ def extractTimestamp(dbName):
     def isExpired(timestamp):
         return now - args.expires > timestamp
 
+    # Delete all documents in the _replicator-database of old backups to avoid that they continue after they are deprecated
+    print("----- Delete backup-documents older than %d seconds -----" % args.expires)
+    for doc in filter(lambda doc: isBackupDb(doc.id) and isExpired(extractTimestamp(doc.id)), replicator.view('_all_docs', include_docs=True)):
+        print("deleting backup document: %s" % doc.id)
+        replicator.delete(doc.doc)
```

Comment:
As discussed yesterday: I did it in a seperate loop, to also delete old replication documents.